client/src/css/Home.css

```diff
@@ -1,28 +1,29 @@
 .home-block-1 {
   background-image: radial-gradient(
-      50% 141%,
+      70% 141% at 0 0,
       rgba(25, 132, 202, 0.6) 1%,
       rgba(105, 62, 158, 0.6) 100%
     ),
     url('/asset/images/splash.jpg');
   background-size: cover;
-  background-position: center;
-  height: 120vh;
+  background-position: top;
+  height: 92vh;
   opacity: 1;
-  padding-left: 7%;
-  padding-bottom: 10%;
-  letter-spacing: 2px;
+  padding-left: 8%;
+  padding-bottom: 6%;
 }
 
 .welcome-text {
-  color: white;
+  color: #fff;
   font-weight: bold;
+  font-size: 60px !important;
```

Comment:
whats the purpose of important

Reply:
do we need it

Reply:
i think so it like forces an overrride
its bad practice but im too lazy to fix it